batch/launch_job.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+import boto3
+import click
+import glob
+import os
+import re
+import subprocess
+import tarfile
+import time
+import yaml
+
+@click.command()
+@click.option("-c", "--config", "config_file", envvar="CONFIG_PATH", type=click.Path(exists=True), required=True,
+              help="configuration file for this run")
+@click.option("-j", "--num-jobs", "num_jobs", type=click.IntRange(min=1), required=True,
+              help="total number of jobs to run in this batch")
+@click.option("-s", "--sims-per-job", "sims_per_job", type=click.IntRange(min=1), required=True,
```

Comment:
nit: we use `-s` to specify the scenario in the other command line tools, may be less confusing to use another alpha